src/altair/helpers.rs

```diff
@@ -68,9 +103,36 @@ pub fn get_next_sync_committee<
         MAX_VALIDATORS_PER_COMMITTEE,
         SYNC_COMMITTEE_SIZE,
     >,
-    _context: &Context,
-) -> Result<()> {
-    Ok(())
+    context: &Context,
+) -> Result<SyncCommittee<SYNC_COMMITTEE_SIZE>> {
+    // Return the next sync committee, with possible pubkey duplicates.
+    let indices = get_next_sync_committee_indices(state, context)?;
+    let public_keys = state
+        .validators
+        .iter()
+        .enumerate()
+        .filter_map(|(i, v)| {
+            if indices.contains(&i) {
+                Some(&v.public_key)
+            } else {
+                None
+            }
+        })
+        .collect::<Vec<_>>();
+    // @dev not sure if the impl of `eth_aggregate_pubkeys` is needed or correct
+    // also, not sure if `expect` is desired here
+    let aggregate_public_key = eth_aggregate_pubkeys(&public_keys)
+        .expect("validator public_keys should be aggregated into aggregate_public_key");
+
+    // @dev ran into trouble here so just cloned() the validator's public_key
+    // couldn't figure out how to properly coerce/convert `public_keys` Vec to ssz_rs::Vector
+    let pks_vector =
+        Vector::<_, SYNC_COMMITTEE_SIZE>::from_iter(public_keys.iter().map(|&pk| pk.clone()));
```

Comment:
Note: fixed with `public_keys.iter().map(|&pk| *pk`

Reply:
Jk nvm lol 
```
cannot move out of `*pk` which is behind a shared reference
move occurs because `*pk` has type `crypto::PublicKey`, which does not implement the `Copy` trait
```